Earthfile

```diff
@@ -95,7 +95,7 @@ build-examples:
   RUN mkdir $TARGET
   DO +GQLMD --options="--homepage data/anilist.md --schema https://graphql.anilist.co/ --base . --link /${TARGET}/default --force --pretty --noPagination --deprecated group"
   RUN mv docs ./$TARGET/default
-  DO +GQLMD --options="--homepage data/groups.md --schema data/schema_with_grouping.graphql --groupByDirective @doc(category|=Common) --base . --link /${TARGET}/group-by --skip @noDoc --index --noTypeBadges --noParentType --noRelatedType --deprecated group"
+  DO +GQLMD --id="schema_with_grouping" --options="--homepage data/groups.md --schema data/schema_with_grouping.graphql --groupByDirective @doc(category|=Common) --base . --link /${TARGET}/group-by --skip @noDoc --index --noParentType --noRelatedType --deprecated group"
```

Comment:
`--id="schema_with_grouping"` nice :star_struck: 

Since `--noTypeBadges` is removed, this will have to be reflected into the homepage of this example [`data/groups.md`](https://github.com/graphql-markdown/graphql-markdown/blob/main/packages/docusaurus/tests/__data__/groups.md?plain=1#L32)